data/resources/ui/content-chat-history.ui

```diff
@@ -26,24 +26,79 @@
           </object>
         </child>
         <child>
-          <object class="GtkScrolledWindow">
-            <property name="vexpand">True</property>
-            <property name="hscrollbar-policy">never</property>
-            <style>
-              <class name="view"/>
-              <class name="chat-history"/>
-            </style>
-            <property name="child">
-              <object class="AdwClampScrollable">
-                <property name="maximum-size">700</property>
-                <property name="tightening-threshold">500</property>
-                <property name="vscroll-policy">natural</property>
+          <object class="GtkOverlay" id="content">
+            <child type="overlay">
+              <object class="GtkRevealer">
+                <property name="transition_type">slide-up</property>
+                <property name="reveal_child" bind-source="ContentChatHistory" bind-property="sticky" bind-flags="sync-create|invert-boolean"/>
+                <property name="valign">end</property>
+                <property name="halign">end</property>
+                <child>
+                  <object class="GtkOverlay">
+                    <property name="width-request">60</property>
+                    <property name="height-request">52</property>
+                    <property name="margin-end">18</property>
+                    <property name="margin-bottom">18</property>
```

Comment:
```suggestion
                    <property name="width-request">60</property>
                    <property name="height-request">52</property>
                    <property name="margin-end">18</property>
                    <property name="margin-bottom">18</property>
```

I would define all of this in css, but on the button instead of the overlay and only via margins. This way we make the button to define everything about this widget: the margins to the bottom left, the spacing with the label on the top and the general sizing of the entire widget. Also, a bit of padding would be good too:

```css
.scroll-to-bottom-button {
  margin: 12px 18px 18px;
  padding: 3px;
}
```